templates/cluster-template-capd.yaml

```diff
@@ -115,8 +115,8 @@ spec:
     spec:
       extraMounts:
         - name: containerd-sock
-          containerPath: unix:///run/containerd/containerd.sock
```

Comment:
I'm not sure if this issue is down to some difference in image or K8s version as I'm fairly certain this template worked previously. :thinking:

Reply:
good catch, i'm not sure when it stopped working but i notice that i don't use it in my [personal manifests](https://github.com/elmiko/capi-hacks/blob/devel/kubemark/kubemark-workload-md0.yaml#L44-L48)